### PR TITLE
add gdc_from_graph setting and mapping files

### PR DESCRIPTION
**Pull request by lindaxiang**

### CODE REVIEW

File: es-models/gdc_from_graph/case.mapping.yaml [hunk 1]
```diff
@@ -0,0 +1,2010 @@
...
+                type: keyword
+              relative_with_cancer_history:
+                type: keyword
+              state:
+                type: keyword
+              submitter_id:
+                type: keyword
+              updated_datetime:
+                type: keyword
+          files:
```
Comment (junjun-zhang) [inline es-models/gdc_from_graph/case.mapping.yaml]: I am surprised to see another `files` node here, it's like: `case.files.cases.files`. We should only have `case.files.cases`. Checked on `case` document, there is not such fourth level of `files`.

Should remove this in the mapping and try load JSON docs into ES in `strict` mode, if loading passed, it's safe to remove this `files` node.

File: es-models/gdc_from_graph/file.mapping.yaml [hunk 2]
```diff
@@ -0,0 +1,1290 @@
...
+            type: keyword
+          relative_with_cancer_history:
+            type: keyword
+          state:
+            type: keyword
+          submitter_id:
+            type: keyword
+          updated_datetime:
+            type: keyword
+      files:
```
Comment (junjun-zhang) [inline es-models/gdc_from_graph/file.mapping.yaml]: Similar to what's commented above in `case` index, this `file.cases.files` node is not need. Should be removed.

File: es-models/gdc_from_graph/case.mapping.yaml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+dynamic: strict
```
Comment (junjun-zhang) [inline es-models/gdc_from_graph/case.mapping.yaml]: We don't need `dynamic`

File: es-models/gdc_from_graph/case.mapping.yaml [hunk 4]
```diff
@@ -42,7 +43,7 @@ properties:
         type: keyword
   case_id:
     type: keyword
-    copy_to:
+    copy_to:
```
Comment (junjun-zhang) [inline es-models/gdc_from_graph/case.mapping.yaml]: trailing whitespace should be avoided

File: es-models/gene_centric/gene_centric.mapping.yaml [hunk 5]
```diff
@@ -196,6 +196,8 @@ properties:
                         type: keyword
                       dbsnp_rs:
                         type: keyword
+                      dbsnp_val_status:
```
Comment (dankolbman) [inline es-models/gene_centric/gene_centric.mapping.yaml]: Updates to things not part of the gdc_from_graph should probably be in a separate pr.

Comment (junjun-zhang) [inline es-models/gene_centric/gene_centric.mapping.yaml]: yes, I agree

Comment (lindaxiang) [inline es-models/gene_centric/gene_centric.mapping.yaml]: Good point.
